templates/partials/participant-consent.html.twig

```diff
@@ -46,4 +46,9 @@
             {% endif %}
         </li>
     {% endif %}
+    <li class="list-group-item">
+        <strong>Participant Status</strong>
+        {{ participant.enrollmentStatusV3_2 }}
+        {{ participant.enrollmentStatusV3_2Time|date('m/d/Y') }}
```

Comment:
This will display current date if `enrollmentStatusV3_2Time` field is null. You might need to do a null check here.